conf/examples/telemeter_stage.yaml

```diff
@@ -1,6 +1,4 @@
 ---
 ENV_DATA:
-  # for testing of OCS telemetry integration with stage instance of telemeter
-  # use via `--cluster-conf conf/ocsci/stage_telemeter.yaml`
```

Comment:
I think we need this but instead it should be --ocsci-conf

Reply:
@vasukulkarni @petr-balogh , is it same for monitoring.yaml as well. 
```--ocsci-conf conf/ocsci/monitoring.yaml``` and ```--ocsci-conf conf/ocsci/stage_telemeter.yaml```